packages/sdk/src/types.ts

```diff
@@ -416,7 +428,7 @@ export type OrderFilter = {
 export type OrderFulfillment = {
   __typename?: 'OrderFulfillment';
   carrier?: Maybe<Scalars['String']>;
-  state: Scalars['String'];
+  state: FulfillmentState;
```

Comment:
Aren't these autogenerated?

Reply:
I believe the types in `dist` get generated from src

Reply:
I dont think we have type generation yet

Reply:
correct, we don't, yet